modules/nw-ingress-creating-a-route-via-an-ingress.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * ingress/routes.adoc
+
+[id="nw-ingress-creating-a-route-via-an-ingress_{context}"]
+= Creating a Route via an Ingress
+
+Some ecosystem components have an integration with Ingress resources but not with
```

Comment:
s/Route/route (throughout assembly - we have it capitalized in the glossary, but throughout the documentation it's mostly lower case. So I'm suggesting that we change it to lower case for consistency.)